content/docs/authorization/cli.md

```diff
@@ -802,16 +782,15 @@ karavictl storage list [flags]
 ##### Options inherited from parent commands
 
 ```
-  -f, --admin-token string   Specify the admin token file
-      --config      string   Config file (default is $HOME/.karavictl.yaml)
-      --addr        string   Address of the server (default "localhost")
-      --insecure             Skip certificate validation
+  -f, --admin-token string   Path to admin token file; required
+      --addr                 Address of the CSM Authorization Proxy-Server; required
```

Comment:
This Proxy-Server is different from the other Proxy Server (without dash)

Reply:
Removed the dash for all instances.